python/doc/pyplots/Kent.py

```
@@ -0,0 +1,117 @@
+grid = ot.GridLayout(2, 4)
+grid.setTitle("Kent(kappa, beta, gamma1, gamma2, gamma3)")
+
+kappa_vals = [5.0, 20.0]
+beta_vals = [1.0, None]
+
+fig = plt.figure(figsize=(16, 8))
+index = 0
+factor = 0.3
+
+for kappa in kappa_vals:
+    for j, beta in enumerate(beta_vals):
```

Ruff (0.15.15)

[warning] Loop control variable `j` not used within loop body

Rename unused `j` to `_j`

(B007)